kong/db/dao/init.lua

```diff
@@ -368,7 +368,7 @@ local function check_update(self, key, entity, options, name)
   end
 
   local rbw_entity
-  if read_before_write then
+  if read_before_write then -- this is currently always true
```

Comment:
if it's always true do we need the conditional at all?

Reply:
@hutchic, the value comes from other function. If for some reason that function changes, we don't need to change code here.

Reply:
@bungle appreciate taking it on the careful side, but I agree with Colin on this case. If the idea is to remove the RBW detection logic, then it's better to remove that `read_before_write` result from `process_auto_fields` (which was always a stretch from the function's original task...). Since the criteria for RBW as of this PR is uniquely the DAO context, I don't think it's worth keeping anything related to `read_before_write` in `kong.db.schema.init`.

Reply:
@hishamhm,

Ok I'll change it.